pulumi/infra/config.py

```diff
@@ -21,6 +21,11 @@
 # note: this ${} is interpolated inside Nomad
 HOST_IP_IN_NOMAD: Final[str] = "${attr.unique.network.ip-address}"
 
+# This is equivalent to what "${attr.unique.network.ip-address}" resolves to but is used for cases where variable
+# interpolation is not available. Currently this is used to get the private IP for Nomad so the otel collector can
+# scrape metrics
+LOCAL_HOST_IP: Final[str] = os.environ["LOCAL_HOST_ETH0_IP"]
```

Comment:
Given that this variable is provided in contrast with `HOST_IP_IN_NOMAD`, it may be better to name it something like `HOST_IP_OUTSIDE_NOMAD` (ugh, that name _sucks_ 🤢) to more clearly tie it to the other variable, and to better highlight its intended usage.